src/chocolatey/infrastructure.app/commands/ChocolateyInstallCommand.cs

```diff
@@ -155,6 +155,11 @@ public virtual void configure_argument_parser(OptionSet optionSet, ChocolateyCon
                      "Stop On First Package Failure - stop running install, upgrade or uninstall on first package failure instead of continuing with others. Overrides the default feature '{0}' set to '{1}'. Available in 0.10.4+.".format_with(ApplicationParameters.Features.StopOnFirstPackageFailure, configuration.Features.StopOnFirstPackageFailure.to_string()),
                      option => configuration.Features.StopOnFirstPackageFailure = option != null
                      )
+                 .Add("stoponfirstrebootrequest|stop-on-first-reboot-request|stop-on-first-package-reboot-request",
+                     "Stop On First Package Reboot Request - stop running install, upgrade, or uninstall on first package reboot request instead of continuing with others.  Overrides the default feature '{0}' set to '{1}'.  Available in 0.10.12+.".format_with

```

Comment:
"Exit When Reboot Detected -" and similar messaging that the feature switch will have.